calicoctl/commands/node/run.go

```diff
@@ -102,6 +103,9 @@ Options:
                            and there are no pre-existing Calico IP pools.
      --disable-docker-networking
                            Disable Docker networking.
+     --docker-networking-ifprefix=<IFPREFIX>
+                           Interface prefix to use for the Docker containers.
```

Comment:
Yeah - this needs to be really clear that this only applies to our libnetwork plugin.

@gunjan5 We'll also need a docs update for this.  Can you please raise the issue / PR in the Calico repo and link it here before merging?